packages/contractkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/contractkit",
-  "version": "0.4.18-dev",
+  "version": "0.4.19-dev",
```

Comment:
I think it would make more sense to do this in a different PR, where you also change all the files that consume the dev version of the contractkit kit